src/servers/middlewares/auth-router.ts

```diff
@@ -168,21 +169,36 @@ authRouter.post(
             .status(500)
             .send({ error: "Missing csrf or ory_kratos_session cookie" })
         }
+        const kratosCookieStr = libCookie.serialize(
+          kratosSessionCookie.name,
+          kratosSessionCookie.value,
+          {
+            expires: kratosSessionCookie.expires,
+            maxAge: kratosSessionCookie.maxAge,
+            sameSite: "none",
+            secure: kratosSessionCookie.secure,
+            httpOnly: kratosSessionCookie.httpOnly,
+            path: kratosSessionCookie.path,
+          },
+        )
+        const session = await kratosPublic.toSession({ cookie: kratosCookieStr })
+        const expiresAt = session.data.expires_at
+          ? new Date(session.data.expires_at)
+          : new Date(new Date().setDate(new Date().getDate() + 30))
```

Comment:
should avoid the magic number, or document it with a dedicated const

Reply:
yeah chatgpt wrote that for me lol. it stands for 30 days from now. where should i put the dedicated const?